docker_compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: shopadmin
+      POSTGRES_PASSWORD: testpassword
+      POSTGRES_DB: shop
```

Comment:
We'll get to it later, but keep in mind that you don't want to have your envars in your repo.